app/generated_resources.grd

```diff
@@ -397,7 +397,7 @@ are declared in tools/grit/grit_rule.gni.
           </message>
           <!-- Also update IDS_CONTENT_CONTEXT_OPENLINKOFFTHERECORD_INAPP in google_chrome_strings.grd and chromium_strings.grd. -->
           <message name="IDS_CONTENT_CONTEXT_OPENLINKOFFTHERECORD" desc="The name of the open a link in incognito window command">
-            Open link in inco&amp;gnito window
+            Open link in &amp;private window
```

Comment:
Seemed as good an idea as any to make <kbd>p</kbd> the shortcut

Reply:
This determines which keypress jumps to that menu item on systems which support that?

Reply:
@tomlowenthal that's right